app-embed-example/setup/metabase-setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+echo "seting up $1"
```

Comment:
Since this has been changed to the Site URL, then update the rest, so URLs are correct with the variable replacements.